### PR TITLE
rock paper scissors in progress, couldnt fix bug

### PR DESCRIPTION
**Pull request by melanieanderson1995**

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -3,57 +3,48 @@
...
-    @game = Game.create(player_1, player_2)
+    $game = Game.new(player_1, player_2)
     erb :play_singleplayer
   end
 
   get "/rock_computer" do
-    @player_1_name = $player_1.name
-    @computer_name = $computer.name
-    @game = Game.new
     @computer_move = @game.random_selection
```
Comment (Pi-hils) [inline app.rb:38]: I like the change of names here (computer_name -->computer_move) as it makes it easier to understand.

File: spec/game_spec.rb [hunk 2]
```diff
@@ -17,8 +19,8 @@
 
   describe "#computer_move" do
     it "returns random selection" do
-      allow(mock_computer).to receive(:random_selection).and_return("rock")
-      expect(mock_computer.random_selection).to eq("rock")
+      allow(player_2).to receive(:random_selection).and_return("rock")
```
Comment (Pi-hils) [inline spec/game_spec.rb:22]: nice use of the edge case here. I like how you separated each player and did a separate test on both.

File: lib/game.rb [hunk 3]
```diff
@@ -8,14 +8,6 @@ def initialize(player_1, player_2)
...
-  def self.instance
-    @game
-  end
-
-  def self.create(player_1, player_2)
-    @game = Game.new(player_1, player_2)
-  end
-
   def random_selection
     @choices.sample
```
Comment (Pi-hils) [inline lib/game.rb:12]: Great use of DI here and your code is DRY so well done for that.
